platform/solr/solr-factory/pom.xml

```diff
@@ -114,22 +294,22 @@
                                         <limit>
                                             <counter>INSTRUCTION</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.11</minimum>
+                                            <minimum>0.09</minimum>
```

Comment:
The coverage of `ConfigurationFileProxy` is around 76%.  Coverage of `ConfigurationStore` could be improved.  The other classes need Solr instances to connect to so they are tested as integration tests if possible (embedded and HTTP can but cloud client cannot currently).  Based on that, the coverage will still be low.